### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by Baker0o7** [Hamza417/Inure#34] [General]
Hi sir thank you for such a beautiful app and your hard work waited for long for the first apk release am the first to download it 👋 soon will give you feedback

**Comment by Hamza417** [comment]
@Baker0o7 You're welcome :)
You weren't using Play Store to test the releases?

**Comment by Baker0o7** [comment]
I don't use google 🙌 but I install and tried your first release build much batter progress now 👌
